Source/objects.cpp

```diff
@@ -1020,16 +1020,6 @@ void AddChest(int i, int t)
 void ObjSetMicro(Point position, int pn)
 {
 	dPiece[position.x][position.y] = pn;
-	pn--;
-
-	int blocks = leveltype != DTYPE_HELL ? 10 : 16;
-
-	uint16_t *piece = &pLevelPieces[blocks * pn];
-	MICROS &micros = dpiece_defs_map_2[position.x][position.y];
-
-	for (int i = 0; i < blocks; i++) {
-		micros.mt[i] = SDL_SwapLE16(piece[blocks - 2 + (i & 1) - (i & 0xE)]);

```

Comment:
This is one of the nicest cleanups from removing/gutting the micro map. You wonder what was running through their minds when they wrote this code :D